documentation/docs/pages/developers/nymvpncli.mdx

```diff
@@ -0,0 +1,278 @@
+2024-12-11T11:03:59.982218Z  INFO nym_vpn_account_controller::controller: Received command: UpdateDeviceState
+2024-12-11T11:03:59.982230Z  INFO nym_vpn_account_controller::shared_state: Setting mnemonic state to NotStored
+2024-12-11T11:03:59.982240Z  WARN nym_vpn_account_controller::commands: Returning error: NoAccountStored
+```
+
+Ignore the `NoAccountStored` errors: these will disappear after the next step. **Leave the daemon running and run the following commands in another terminal window** or create an init file for `nym-vpnd`.
+
+## Run VPN
+We have to first store the account we have created online:
+```sh
+./PATH/TO/nym-vpn-cli store-account --mnemonic "<MNEMONIC_FROM_GENERATION_STEP>"
+```
+
+You will see this registration in the daemon logs:
+
+```sh
+2024-12-11T11:04:31.918455Z  INFO grpc_vpnd: ← StoreAccount ()
+2024-12-11T11:04:31.919296Z  INFO nym_vpnd::service::vpn_service: Storing account
+2024-12-11T11:04:31.919531Z  INFO nym_vpn_store::mnemonic::on_disk: Storing mnemonic to: /var/lib/nym-vpnd/mainnet/mnemonic.json
+2024-12-11T11:04:31.920327Z  INFO nym_vpn_account_controller::controller: Received command: UpdateAccountState
+2024-12-11T11:04:31.950720Z  INFO nym_vpn_account_controller::shared_state: Setting mnemonic state to Stored { id: "n1nghj6qnmfww22tq6wyntnf709lr90qjem0uezz" }
+2024-12-11T11:04:34.616249Z  INFO nym_vpn_account_controller::shared_state: Setting account to Registered
+2024-12-11T11:04:34.616363Z  INFO nym_vpn_account_controller::shared_state: Setting account summary to AccountSummary { account: Active, subscription: Active, device_summary: DeviceSummary { active: 0, max: 10, remaining: 10 }, fair_usage: FairUsage { used_gb: None, limit_gb: None, resets_on_utc: Some("2025-01-09 15:43:37.223Z") } }
+2024-12-11T11:04:34.981875Z  INFO nym_vpn_account_controller::controller: Received command: RegisterDevice
+2024-12-11T11:04:35.008575Z  INFO register_device: nym_vpn_account_controller::shared_state: Setting device registration result to InProgress id=09876a3a
+2024-12-11T11:04:35.008611Z  INFO register_device: nym_vpn_account_controller::commands::register_device: Registering device: Device { identity_key: BZWA5MRnEvRYD8WWrH9KULdj2Q1uTssu6idjgWFae9dv } id=09876a3a
+2024-12-11T11:04:36.765850Z  INFO register_device: nym_vpn_account_controller::commands::register_device: Response: NymVpnDevice {
+    created_on_utc: "2024-12-11 11:04:36.432Z",
+    last_updated_utc: "2024-12-11 11:04:36.432Z",
+    device_identity_key: "BZWA5MRnEvRYD8WWrH9KULdj2Q1uTssu6idjgWFae9dv",
+    status: Active,
+} id=09876a3a
+2024-12-11T11:04:36.765998Z  INFO register_device: nym_vpn_account_controller::commands::register_device: Device registered: BZWA5MRnEvRYD8WWrH9KULdj2Q1uTssu6idjgWFae9dv id=09876a3a
+```
+
+You can then connect `nym-vpnc` (in this case, with 2 hop wireguard mode enabled):
+```sh
+./PATH/TO/nym-vpn-cli nym-vpnc connect --enable-two-hop
```

Comment:
I would consider to:
```
import { VarInfo } from 'components/variable-info.tsx';
```

Then put 
```
<VarInfo />
```
Somewhere on top and use the PATH cnvention like it's noted there. 

https://nymtech.net/docs/operators/variables

Which is `<PATH_TO>`